commands/up.go

```diff
@@ -64,20 +74,87 @@ func preRunUp(cmd *cobra.Command, args []string) error {
 }
 
 func upHandler(cmd *cobra.Command, args []string) error {
-	if err := runBuild(cmd, args); err != nil {
+	handler := upRunner(cmd, args)
+	if err := handler(); err != nil {
 		return err
 	}
-	fmt.Println()
-	if !skipPush {
-		if err := runPush(cmd, args); err != nil {
+
+	if watch {
+		logger := log.New(os.Stderr, "", log.LstdFlags)
+		logger.SetFlags(log.Ldate | log.Ltime | log.LUTC)
+
+		watcher, err := fsnotify.NewWatcher()
+		if err != nil {
 			return err
 		}
-		fmt.Println()
-	}
-	if !skipDeploy {
-		if err := runDeploy(cmd, args); err != nil {
+		defer watcher.Close()
+
+		err = filepath.Walk("./", func(path string, info os.FileInfo, err error) error {
```

Comment:
Perhaps also inline this:

if err:= ; err!=nil {

}

Reply:
Resolved